impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.views.conf.js

```diff
@@ -60,8 +60,6 @@ define([
             plotFrameVisible: false,
             plotSizeMin: {width: 600, height: 300},
```

Comment:
yes. I know i can override this, but knowing this is a platform wide config, defining sizes for the charts might be too strong